### ISSUE TITLE
Narratives

### ISSUE THREAD

**Issue by lliskovoi** [inclusive-design/financial-inclusion#157]
- Journey maps are missing alt text (not marked as decorative).
- Can we tighten spacing in impact/effort diagrams and lengthen the line length of text in the Edge User Deep dive list to fit those sections on the same page as the heading and blurb that precedes it?
